services/121-service/src/registration/dto/bulk-import.dto.ts

```diff
@@ -25,7 +25,7 @@ const fspArray = Object.values(FinancialServiceProviders).map((item) =>
   String(item),
 );
 const languageArray = Object.values(LanguageEnum).map((item) => String(item));
-export class BulkImportDto {
+class BulkImportDto {
```

Comment:
This should not have Dto suffix if it's not exported and therefore not used in controller any more right, according to guidelines? And it can lose the swagger decorators (and become an interface?). Related to this there are more things to change in this file I think. ImportResult should have Dto suffix. Split across files. Etc.

Reply:
Agreed. But feel like this is a bit out of scope for this work. I just removed the export here because our knip check was complaining about this export